.github/workflows/rcBuild.yml

```diff
@@ -121,3 +119,18 @@ jobs:
               "npm-package": "sfdx-cli",
               "version": "${{ github.event.release.tag_name }}"
             }
+
```

Comment:
just-nuts now becomes a "hard dependency" of this job.   It'll fail the entire build if the just-nuts fail.

I think that's fine and probably good as it increases visibility/urgency of those failures.  But we had originally tried to avoid doing that.

Reply:
Yea, I considered this too. I do think it is very easy for us to miss just-nuts failures on releases. This will be more important once we have nightlies running. We want to surface nut failures so that we can catch regressions before getting automatically promoted to `latest-rc`. It will also encourage us to further improve some flakiness 😅